### PR TITLE
fixed typos and made the readme with headers

### PR DESCRIPTION
**Pull request by Beosti**

### CODE REVIEW

**Comment by Beosti** [comment]
you should always have the modid at one place and never write that out again

**Comment by Beosti** [comment]
Instead of doing `public static final RegistryObject<Item> KATANA_OF_WIND = ITEMS.register("katana_of_wind",
            () -> new KatanaOfWindItem(KatanaOfWindItem.KATANA_OF_WIND, 2, -1.8f, new Item.Properties()
                    .tab(ModCreativeModeTabs.ECLIPSE_TAB)));` you can just do `    public static final RegistryObject<Item> KATANA_OF_WIND = ITEMS.register("katana_of_wind", KatanaOfWindItem::new); if you do in your actual item:
`    public KatanaOfWindItem() {
        super(KATANA_OF_WIND, 2, -1.8f, new Item.Properties().tab(ModCreativeModeTabs.ECLIPSE_TAB));
    }` instead of this `    public KatanaOfWindItem(Tier pTier, int pAttackDamageModifier, float pAttackSpeedModifier, Properties pProperties) {
        super(pTier, pAttackDamageModifier, pAttackSpeedModifier, pProperties);
    }`

tl;dr you should try and 'contain' your item stuff within your item class

**Comment by Beosti** [comment]
well the format of the writing is fucked but just read it
